client/app/telecom/telephony/billingAccount/orderAlias/telecom-telephony-billing-account-orderAlias.service.js

```diff
@@ -178,14 +178,13 @@ angular.module("managerApp").service("TelecomTelephonyBillingAccountOrderAliasSe
      */
     this.getForeignCountries = function () {
         return this.getUser()
-            .then(function (user) {
+            .then(function () {
                 return OvhApiTelephony.Lexi().schema().$promise.then(
```

Comment:
you can use `TelephonyMediator.getApiModelEnum("telephony.NumberCountryEnum")` to fetch the right enum ;-)